source/docs/advanced/graphql/dataloaders-and-cache-invalidation.md

```diff
@@ -79,6 +79,10 @@ The second one is more specific to GraphQL. To understand its gain, we should co
 
 If the product `PANT-01` was an upsell of all other products in the `pants` category, the inventory API would be requested again when resolving the `upsells.qty` field. Query-level caching prevents those extra-calls, by reusing the response from `products.qty` that has already been fetched previously to resolve data.
 
+### Clearing the cache
```

Comment:
**nitpick(non-blocking):** the use case here is to do it manually from the remote platform, it is not clear from the title

```suggestion
### Manually clearing the cache
```